docker-compose.yml

```
@@ -18,3 +18,17 @@ services:
     environment:
       ADMINER_PLUGINS: 'tables-filter'
       ADMINER_DEFAULT_SERVER: 'postgres'
+
+  mysql:
+    image: mysql:5
+    environment:
+      MYSQL_ROOT_PASSWORD: 'secret'
+      MYSQL_DATABASE: 'adp-dev'
+      MYSQL_USER: 'adp'
+      MYSQL_PASSWORD: 'secret'
+    command: --init-file /data/application/init.sql
+    volumes:
+      - ./mysql:/var/lib/mysql
+      - ./resources/adp-dev.sql:/data/application/init.sql
+    ports:
+      - 0.0.0.0:3306:3306
```

```suggestion
      - 127.0.0.1:3306:3306
```

Do not expose MySQL service publicly.